1.Basic-Video-Chat/Basic-Video-Chat/ViewController.swift

```diff
@@ -9,9 +9,6 @@
 import UIKit
 import OpenTok
 
-let kWidgetHeight = 240
-let kWidgetWidth = 320
```

Comment:
I replace it just to try to make this area as clean as possible so users can focus on `Fill the following variables using your own Project info` block.